src/helm/benchmark/window_services/flan_t5_window_service.py

```diff
@@ -0,0 +1,30 @@
+from .encoder_decoder_window_service import EncoderDecoderWindowService
+from .tokenizer_service import TokenizerService
+
+
+class FlanT5WindowService(EncoderDecoderWindowService):
+    def __init__(self, service: TokenizerService):
+        super().__init__(service)
+
+    @property
+    def max_sequence_length(self) -> int:
+        """Return the max sequence length."""
+        # From https://arxiv.org/pdf/1910.10683.pdf, "we use a maximum sequence length of 512".
```

Comment:
This comment is for T5. Can we update this?

Reply:
Removed the stale links. I checked on Hugging Face AutoTokenizer that both of these are correct.